staging/src/k8s.io/client-go/tools/reference/ref_test.go

```diff
@@ -37,29 +37,31 @@ func TestGetReferenceRefVersion(t *testing.T) {
 	tests := []struct {
 		name               string
 		input              *TestRuntimeObj
+		groupVersion       schema.GroupVersion
 		expectedRefVersion string
 	}{
 		{
 			name: "api from selflink",
```

Comment:
does the testcase name need to change if this is no longer extracting from selflinks?

Reply:
done